tests/model_registry/python_client/test_model_registry_creation_oauth.py

```diff
@@ -0,0 +1,77 @@
+import pytest
+from typing import Self
+from simple_logger.logger import get_logger
+from pytest_testconfig import config as py_config
+from semver import Version
+from utilities.infra import get_product_version
+from utilities.constants import DscComponents
+from tests.model_registry.constants import MODEL_NAME, MODEL_DICT
+from model_registry import ModelRegistry as ModelRegistryClient
+
+LOGGER = get_logger(name=__name__)
+MINVER = Version.parse(version="2.21.0")
+
+
+@pytest.mark.parametrize(
+    "updated_dsc_component_state_scope_class, model_registry_client",
+    [
+        pytest.param(
+            {
+                "component_patch": {
+                    DscComponents.MODELREGISTRY: {
+                        "managementState": DscComponents.ManagementState.MANAGED,
+                        "registriesNamespace": py_config["model_registry_namespace"],
+                    },
+                },
+            },
+            {"service_fixture": "model_registry_instance_oauth_service"},
+            id="oauth_proxy",
+        ),
+    ],
+    indirect=True,
+)
+@pytest.mark.usefixtures("updated_dsc_component_state_scope_class")
+class TestModelRegistryCreationOAuth:
+    """
+    Tests the creation of a model registry with OAuth proxy configuration.
+    Jira ID: RHOAIENG-26194
+    """
+
+    @pytest.mark.smoke
+    def test_registering_model_with_oauth(
+        self: Self,
+        admin_client,
+        model_registry_client: ModelRegistryClient,
+    ):
+        if py_config["distribution"] == "downstream" and get_product_version(admin_client=admin_client) < MINVER:
+            pytest.skip("Skipping test for RHOAI < 2.21")
+
```

Comment:
Is it not valid for odh? I still feel this should be moved out of this PR. We can mark it as smoke, after we get a chance to discuss and push for use of right tags for jenkins jobs.

Reply:
It is valid but we don't know from which ODH release, in ODH right now it won't ever be skipped. AFAIK we don't have automated runs that include opendatahub-tests against ODH so it shouldn't be a problem